src/javascript/upgrade-dependencies.ts

```diff
@@ -388,6 +391,21 @@ export class UpgradeDependencies extends Component {
     return dependencies.map((d) => d.name);
   }
 
+  /**
+   * Projen only uses npm-check-updates to upgrade dependencies when either the version is omitted, or set to "*".
+   * Otherwise, the exact version selected is placed in the package.json file and upgrading is handled through the package manager
+   * rather than npm-check-updates.
+   *
+   * @param version semver from DependencyCoordinates.version, may be undefined
+   * @returns whether the version is the default versioning behavior
+   */
+  private packageIsUsingDefaultVersioning(
```

Comment:
Let's give this a more descriptive name
```suggestion
  private isProjenManagedVersion(
```